.github/workflows/build-cmake.yaml

```
@@ -0,0 +1,43 @@
+name: "[internal] Build cmake"
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * *"
+
+jobs:
+  build-cmake:
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        runner-info: [
```

```suggestion
        arch: [grayskull, wormhole_b0]
```

no need for `runner-info` block